preview/app/controllers/FaciaDraftController.scala

```diff
@@ -32,4 +33,21 @@ class FaciaDraftController(
   override def canRender(path: String): Boolean = ConfigAgent.getPathIds.contains(path)
 
   override def canRender(item: ItemResponse): Boolean = indexController.canRender(item)
+
+  override def renderFrontShowcase(path: String): Action[AnyContent] =
+    Action.async { implicit request =>
+      frontJsonFapi.get(path, liteRequestType).map {
+        case Some(faciaPage: PressedPage) =>
```

Comment:
Not sure, but I think this could be written as:
`case Some(faciaPage: PressedPage) if TrailsToShowcase.isShowcaseFront(faciaPage) =>`